src/components/SearchResults.js

```diff
@@ -1,13 +1,17 @@
 // @flow
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Tabs, Tab } from 'react-bootstrap';
```

Comment:
Is there a reason to use the bootstrap <Tabs> and <Tab> directly instead of using a <TabPanel> component (the same thing that's used when there are child documents from a join...)?

Reply:
@luigivaldez I'm not sure what the pros and cons are of using the react-bootstrap components vs the SUIT components (I assume you're talking about TabPanel/TabInfo? Is the "standard" to use those over the bootstrap ones?